superduperdb/ext/openai/model.py

```diff
@@ -151,6 +153,9 @@ class OpenAIChatCompletion(OpenAI):
     takes_context: bool = True
     prompt: str = ''
 
+    def __post_init__(self):
```

Comment:
We just need to add a default str encoder for ibis, do not need in mongodb

Reply:
yeah I just realised after talking with @blythed

Reply:
right but at the level of the models, neither `ibis` or `mongodb` exist.

Reply:
This could be handled with `Model.pre_create`. 

```python
def pre_create(self, db):
    if isinstance(db.databackend, IbisBackend) and self.Encoder is None:
        self.encoder = dtype('str')
```